programs/system/Cargo.toml

```diff
@@ -22,6 +22,7 @@ solana-type-overrides = { workspace = true }
 [dev-dependencies]
 assert_matches = { workspace = true }
 solana-compute-budget = { workspace = true }
+solana-feature-set = { workspace = true }
```

Comment:
Oh turns out the tests do use this via a macro

Reply:
Ah darn, this does constitute a breaking change, since `with_mock_invoke_context!` is requiring `solana_feature_set` to be present at https://github.com/anza-xyz/agave/blob/c7e44c1389e01b576b1180387db20971eb1c8d3c/program-runtime/src/invoke_context.rs#L711

Since it's only meant to be used internally for native programs, I think we can get away with it.